pkgs/by-name/me/meson/package.nix

```diff
@@ -21,13 +21,13 @@ let
 in
 python3.pkgs.buildPythonApplication rec {
   pname = "meson";
-  version = "1.4.1";
+  version = "1.5.0";
 
   src = fetchFromGitHub {
     owner = "mesonbuild";
     repo = "meson";
     rev = "refs/tags/${version}";
-    hash = "sha256-RBE4AUF5fymUA87JEDWtpUFXmVPFzdhZgDI7/kscTx4=";
```

Comment:
We are not using an SRI hash here?

Reply:
I think this is already using SRI hash (`hash = "<algo>-<hash>";`), non-SRI hash looks like `sha{1,256,512} = "<hash>";`

Relavant:
- https://github.com/NixOS/nixpkgs/blob/521d48afa9ae596930a95325529df27fa7135ff5/pkgs/build-support/fetchurl/default.nix#L64-L65
- https://github.com/NixOS/nixpkgs/blob/521d48afa9ae596930a95325529df27fa7135ff5/pkgs/build-support/fetchurl/default.nix#L67-L72